app/admin/contributions.rb

```diff
@@ -68,6 +68,17 @@
     end
   end
 
+  form do |f|
+    f.semantic_errors
+    inputs I18n.t('contribution_details') do
+      input :user
+      input :company
```

Comment:
company is an open field only when the logged admin is super admin, if not, will use the admin company_id